tests/experimental/nn/test_multiprocess_pipe.py

```diff
@@ -154,8 +156,59 @@ def update(devices):
     torch.random.manual_seed(3)
     criterion = DistributedLoss(torch.nn.MSELoss)
     x = torch.randn(8, 4).to(device)
-    model = [("linear1", nn.Linear, (4, 4), {}), ("relu", nn.ReLU, (), {})]
-    pipe = MultiProcessPipe(model, balance=[1, 1], chunks=4, devices=devices[:2])
+    model = [PipelineModule(nn.Linear, (4, 4), {}), PipelineModule(nn.ReLU, (), {})]
```

Comment:
Are we eventually thinking of creating helpers that can extract this info from a user's model? A user will have to rewrite their model to use this API

Reply:
I am thinking of this API as a relatively low level API. We can have some sort of FX-tracing to extract the graph for the user. Or we can have a sharder utility that shards a layer and inserts them in the graph. But these ideas need a bit more work.